test/cli.js

```diff
@@ -303,6 +310,47 @@ describe('CLI', () => {
         });
     });
 
+    it('starts the inspector with --inspect', { skip: (NODE_MAJOR <= 4) }, (done) => {
```

Comment:
I believe the inspector first appeared in v6.3.0. I think the skip condition should be "good enough" for CI, but could break if someone was running on v6.1.0 for example.